src/main/psxsdk/libgte/reg03.c

```diff
@@ -8,29 +8,31 @@ void SetVertex1(SVECTOR* vector) { gte_ldv1(vector); }
 void SetVertex2(SVECTOR* vector) { gte_ldv2(vector); }
 
 void SetVertexTri(SVECTOR* vector1, SVECTOR* vector2, SVECTOR* vector3) {
-    gte_ldv0(vector1);
-    gte_ldv1(vector2);
-    gte_ldv2(vector3);
+    gte_ldv3(vector1, vector2, vector3);
```

Comment:
This is a small update for my last change in #1598, since I didn't realise there was already a `gte_ldv3` defined here. I've updated to use that.